types/primus/index.d.ts

```diff
@@ -192,5 +193,6 @@ declare namespace Primus {
         on(event: 'reconnect timeout' | 'reconnect failed', handler: (err: Error, opts: ReconnectEventOpts) => void): this;
         on(event: 'data', handler: (message: any) => void): this;
         on(event: 'error', handler: (err: Error) => void): this;
+        open(): void;
```

Comment:
@sheetalkamat IIRC correctly it returns `this` and not undefined.

Reply:
Yeah, you're right!

Reply:
#45063 I made a pull request fixing it, feel free to review it 🙂